.github/CODEOWNERS

```
@@ -57,9 +58,12 @@
 /editor/inspector/                            @godotengine/docks
 /editor/scene/                                @godotengine/core
 /editor/scene/2d/                             @godotengine/2d-editor
+/editor/scene/2d/physics                      @godotengine/2d-editor @godotengine/physics
 /editor/scene/3d/                             @godotengine/3d-editor
+/editor/scene/3d/physics                      @godotengine/3d-editor @godotengine/physics
 /editor/scene/gui/                            @godotengine/usability @godotengine/gui-nodes
```

```suggestion
/editor/scene/gui/                            @godotengine/gui-nodes
```
Optionally, realized that I had just copied this one from the `/editor/gui/` section and it might not match as well now